### ISSUE TITLE
Shadow Hop

### ISSUE THREAD

**Issue by iRollSolo** [euthyphro666/proto-ninja#13]
When in shadows, the character fades out. 
When faded, the player can press a direction and a button to teleport to the nearest shadow in that direction.

**Comment by swoobie** [comment]
I think we'll want to remove the water or darken the shadows for this one. The water and shadows look very similar right now

![image](https://user-images.githubusercontent.com/9885354/74407695-637ad580-4de7-11ea-8fa3-f4135b2bad5c.png)

**Comment by sentient-soup** [comment]
I agree, I think the level would have to be more intentionally designed to allow for this to be comfortable.
